### DISCUSSION TITLE
How simple is to setup my DAO with Snapshot ?

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#1191] [Q&A]
Vote for an answer below

**Comment by metrox-eth** [comment]
Basically all you need to setup your DAO is:

-An ENS name on mainnet to create a space on Snapshot
-A governance token, almost any token can be used (even NFTs), we support numerous chains as well.
-A Multi-sig (Gnosis Safe)
-You may want to use Discord or Discourse, Twitter and so on to build your community and raise awareness on your project.

**Reply by luisotravez** [reply]
Hi! How can I know which chains are available?

**Reply by metrox-eth** [reply]
Here we go: https://snapshot.org/#/networks

**Reply by shakeabi** [reply]
Hi, may I ask why the necessity of a Gnosis-safe. Afaik from the docs, we can use a Gnosis safe to vote instead of a personal wallet like metamask.

**Reply by metrox-eth** [reply]
I was referring to the bare minimum to setup a DAO, and here the Gnosis Safe was cited as the treasury vault. But yes you can make the Safe the controller of your Snapshot space if it's relevant for you, vote on proposals with your safe, and last but not least, use the Safesnap plugin to write the outcome of a proposal on-chain or execute a transaction. A Safe is an important piece of DAO tooling.

**Reply by shakeabi** [reply]
Thanks for the detailed explanation. :)
